diff --git a/dist/providers/openai-codex-responses.js b/dist/providers/openai-codex-responses.js
index 76efb05bf2724ce15af5fac82c2518e046a62db7..2c5158a56994c3d491469604cd9f231cd904e517 100644
--- a/dist/providers/openai-codex-responses.js
+++ b/dist/providers/openai-codex-responses.js	
@@ -201,7 +201,7 @@ function buildRequestBody(model, context, options) {
     });
     const body = {
         model: model.id,
-        store: false,
+        store: true,
         stream: true,
         instructions: context.systemPrompt,
         input: messages,
diff --git a/dist/providers/openai-responses.js b/dist/providers/openai-responses.js
index 3bf172acee9286eb71fb40c067b303ea96563486..f72f6ae106f13052ce4e25273d2c3145b3c0cc98 100644
--- a/dist/providers/openai-responses.js
+++ b/dist/providers/openai-responses.js	
@@ -139,7 +139,7 @@ function buildParams(model, context, options) {
         stream: true,
         prompt_cache_key: cacheRetention === "none" ? undefined : options?.sessionId,
         prompt_cache_retention: getPromptCacheRetention(model.baseUrl, cacheRetention),
-        store: false,
+        store: true,
     };
     if (options?.maxTokens) {
         params.max_output_tokens = options?.maxTokens;
diff --git a/package.json b/package.json
index 7eec63628cf457c3abd7b40cfb521889abbb81aa..f678ab7029247f5b1715423950cbc07fc4946581 100644
--- a/package.json
+++ b/package.json	
@@ -17,7 +17,8 @@
 		"./bedrock-provider": {
 			"types": "./bedrock-provider.d.ts",
 			"import": "./bedrock-provider.js"
-		}
+		},
+		"./dist/*": "./dist/*"
 	},
 	"bin": {
 		"pi-ai": "./dist/cli.js"
